.github/workflows/lint.yml

```diff
@@ -60,3 +60,23 @@ jobs:
 
         export LANG=en-us
         make pylint
+  
+  pylint_site_packages:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Lint with pylint
+      run: |
+        sudo apt-get install libkrb5-dev
+        pip install -r requirements/devel.txt
```

Comment:
we don't want to be running against all devel dependencies, only against runtime ones, see what gets installed for bandit. should be the same here.

Reply:
The bandit job doesn't have pylint installed as a dependency. Not sure what I should do there other than add pip install pylint-django into the commands directly

Reply:
install only the runtime dependencies. We don't want to analyze all devel tools which don't make it into production.

Reply:
This is still unresolved, see https://github.com/kiwitcms/Kiwi/blob/master/.github/workflows/security.yml#L45

Reply:
FTR running locally against a venv with only the runtime dependencies doesn't crash for me while running against all devel dependencies does. There are almost 3 times difference the number of Python files between the two environments. Fixing this will probably help.